formats/help/publisher/examples/help_page.json

```diff
@@ -0,0 +1,32 @@
+{
+  "content_id": "464c5c16-dcdb-4ed0-9ba7-341ee652038a",
+  "title": "About GOV.UK",
+  "base_path": "/help/about-govuk",
+  "description": "GOV.UK is the best place to find government services and information for citizens and businesses, guidance for professionals, government policy",
+  "schema_name": "help",
+  "document_type": "help_page",
+  "need_ids": [],
+  "public_updated_at": "2014-11-12T12:25:24.000+00:00",
+  "publishing_app": "publisher",
+  "rendering_app": "frontend",
+  "routes": [
+    {
+      "path": "/help/about-govuk",
+      "type": "exact"
+    },
+    {
+      "path": "/help/about-govuk.json",
+      "type": "exact"
+    }
+  ],
+  "redirects": [],
+  "update_type": "major",
+  "change_note": null,
+  "links": {
+    "external_related_links": []
```

Comment:
We've seemingly changed to a new help page in our examples along the way, but I see your point. Just checked, and the `ordered related links` now match those rendered in production.